src/test/ui/directory_ownership/macro-expanded-mod.stderr

```diff
@@ -1,8 +1,8 @@
 error: Cannot declare a non-inline module inside a block unless it has a path attribute
-  --> $DIR/macro-expanded-mod.rs:22:15
+  --> $DIR/macro-expanded-mod.rs:14:28
    |
-LL |     mod_decl!(foo); //~ ERROR Cannot declare a non-inline module inside a block
-   |               ^^^
```

Comment:
So should we not land this?

Reply:
I'm not sure. Looking at the diff, it's not obviously _wrong_, it's just misleading because this error is caused by _where_ the macro was called. I'll take a look later in the week to see if I can avoid this one change in output.

Reply:
I don't think it's a blocker, but I'd like to try fixing it.

Reply:
Fixed.